packages/storybook-blocks/.storybook/theme.js

```diff
@@ -4,7 +4,7 @@ import logo from './logo.png';
 export default create({
   base: 'light',
   brandImage: logo,
-  brandUrl: 'https://style.monday.com', // TODO set brand url
+  brandUrl: 'https://mondaycom.github.io/vibe-storybook-components',
```

Comment:
Do we publish new versions there? I'm not sure, I think we only publish core

Reply:
Oh yeah we do but we use `https://style.monday.com/storybook-blocks`, I'll change it, good catch

Reply:
had no idea, lol
we should find how it's done and apply this logic for the /v2 docs (can also be nice to create one for /v1 on the way, one-time deploy)